webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPDirectiveRepository.scala

```diff
@@ -1025,8 +1030,8 @@ class WoLDAPDirectiveRepository(
     for {
       con             <- ldap
       activeTechnique <- getUPTEntry(con, uactiveTechniqueId, A_ACCEPTATION_DATETIME).notOptional(s"Active technique with id '${uactiveTechniqueId.value}' was not found")
+      oldAcceptations <- ZIO.fromEither(mapper.unserializeAcceptations(activeTechnique(A_ACCEPTATION_DATETIME).getOrElse("")))
```

Comment:
what's the semantic of moving this line?

Reply:
It's now in an IOResult since unserializeAcceptation is now a PureResult see https://github.com/Normation/rudder/pull/3247/files#diff-392a77504d47edf5cb77a4792164ad3b4e876b1229638ef9eb71bbcd31c48656R493